.github/workflows/ci-front.yaml

```diff
@@ -11,12 +11,12 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  STORYBOOK_BUILD_CACHE_KEY: storybook-build-depot-ubuntu-22.04-8-runner
+  STORYBOOK_BUILD_CACHE_KEY: storybook-build-depot-ubuntu-24.04-8-runner
```

Comment:
I wonder if this could be infered from the job name and runner name
RUNNER_NAME

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables

Reply:
Unless I'm mistaken the [runner-context](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#runner-context) is only accessible from a spawned job.
Which means that this env high level statement won't have access to it

Reply:
But we could still centralized the current runner using defaulted non required `inputs`, and also consume this input to build the `storybook-cache-key` dynamically

Reply:
we don't have to store this in an env variable if we can compute it determinisitcally. I feel we only need "storybook-build" to be passed as the cache-key, and then runner can be computed within cache composite action

Reply:
like a suffix that's automatically computed

Reply:
I think we want to keep the cache tied to a runner in any case. Lesson learned